### PR TITLE
switch to use newly-published S3 Resource from Docker Hub

### PR DESCRIPTION
**Pull request by afeld**
Reverts part of https://github.com/18F/concourse-compliance-testing/pull/17, now that the Slack resource with the new config we need is released. Verified locally, which posted https://18f.slack.com/archives/compliance-toolkit/p1458683332000487.

### CODE REVIEW

File: pipelines/zap.yml [hunk 1]
```diff
@@ -2,12 +2,11 @@
...
-
-# Previously we defined the slack notification as a resource, but the one on
-# docker hub (https://hub.docker.com/r/nopik/slack-notification-resource/) is
-# outdated. Instead, we ae using cloud.gov's built-in version
-# (https://github.com/18F/slack-notification-resource-boshrelease).
 resource_types:
+- name: slack-notification
+  type: docker-image
+  source:
+    repository: cfcommunity/slack-notification-resource
```
Comment (afeld) [inline pipelines/zap.yml:9]: Strong feelings about using a specific tag vs. not?

https://hub.docker.com/r/cfcommunity/slack-notification-resource/tags/

**Comment by DavidEBest** [comment]
@afeld No particularly strong feelings. I'd probably stick with 1.0.0.

File: pipelines/zap.yml (hunk 1 above)
Comment (ctro) [inline pipelines/zap.yml:9]: No super strong feelings here either.  Though I guess if I must choose I'd rather be forced to remember to upgrade than run the risk of breaking changes.  i.e. specify the current version.
Could also argue that leaving version out is nice cause we'll pick up new security patches.

**Comment by afeld** [comment]
Updated!

**Pull request merged**